sormas-backend/src/main/java/de/symeda/sormas/backend/common/CronService.java

```diff
@@ -289,4 +290,11 @@ public void archiveTravelEntry() {
 	public void deleteExpiredSpecialCaseAccesses() {
 		specialCaseAccessFacade.deleteExpiredSpecialCaseAccesses();
 	}
+
+	@Schedule(hour = "2", minute = "35", persistent = false)
+	public void syncUsersFromAuthenticationProvider() {
+		if (userFacade.isSyncEnabled() && featureConfigurationFacade.isFeatureEnabled(FeatureType.AUTH_PROVIDER_TO_SORMAS_USER_SYNC)) {
```

Comment:
these two elements can never be both true so the code within if clause will never run